osa/scripts/calibrationsequence.py

```diff
@@ -182,15 +182,15 @@ def drs4_pedestal(
     -------
     Return code
     """
+    if options.simulate:
+        return 0
+
```

Comment:
I actually moved the simulate option after building the commands for the scripts so I could test them. But I think it does not matter in the end.

Reply:
I had to move it before the `get_input_file` call to avoid raised errors for non existing files.

Reply:
OK, fine with me as it is.